app/Controllers/Http/StaticController.js

```diff
@@ -12,61 +12,17 @@ class StaticController {
   }) {
     return response.send({
       needToWaitToProceed: false,
-      isSpellcheckerPremiumFeature: false,
-      isSpellcheckerIncludedInCurrentPlan: true,
       isServiceProxyEnabled: true,
-      isServiceProxyIncludedInCurrentPlan: true,
-      isServiceProxyPremiumFeature: true,
-      isWorkspacePremiumFeature: false,
       isWorkspaceEnabled: true,
       isAnnouncementsEnabled: true,
       isSettingsWSEnabled: false,
       isServiceLimitEnabled: false,
       serviceLimitCount: 0,
-      isCommunityRecipesPremiumFeature: false,
-      isCommunityRecipesIncludedInCurrentPlan: true,
-      isCustomUrlIncludedInCurrentPlan: true,
       isMagicBarEnabled: true,
-      isTeamManagementIncludedInCurrentPlan: true,
       isTodosEnabled: true,
-      isTodosIncludedInCurrentPlan: true,
-      defaultTrialPlan: 'franz-pro-yearly',
-      subscribeURL: 'https://ferdium.org',
```

Comment:
This line also exists in https://github.com/ferdium/ferdium-app/blob/8ab1d10fcc1a120c031de7180019271c1c012335/src/internal-server/app/Controllers/Http/StaticController.js#L15

we should probably also delete it in the main repo

Reply:
Done - thanks for pointing this out. Both the code blocks are now reconciled.